### PR TITLE
Debug tooltip compat audit

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: surf-1533

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Built with commit d1ecb2f545f7588275818eeeaef5cb44f17ae82e

https://app.netlify.com/sites/helix-ui/deploys/5c3cf499afe36300085c8a70

**Comment by CITguy** [comment]
Edge18 and IE11 on Windows 10 LGTM

**Comment by CITguy** [comment]
Dev LGTM

File: src/helix-ui/mixins/Positionable.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const Positionable = (superclass) => {
             this.addEventListener('close', this.__onClose);
 
             if (this.open) {
-                this.__addOpenListeners();
+                this.$emit('open');
```
Comment (CITguy) [inline src/helix-ui/mixins/Positionable.js]: Should fix #410

File: src/helix-ui/mixins/Positionable.js [hunk 2]
```diff
@@ -100,8 +100,12 @@ export const Positionable = (superclass) => {
          * @type {HTMLElement}
          */
         get relativeElement () {
+            if (!this.isConnected) {
+                return;
+            }
```
Comment (CITguy) [inline src/helix-ui/mixins/Positionable.js]: Should fix #411

**Pull request merged**